packages/rspack/src/index.ts

```diff
@@ -34,7 +34,8 @@ export {
 	EntryPlugin,
 	ExternalsPlugin,
 	EnableChunkLoadingPlugin,
-	HotModuleReplacementPlugin
+	HotModuleReplacementPlugin,
+	optimize
```

Comment:
```suggestion
	LimitChunkCountPlugin
```

Reply:
Ohh, wrong suggested change, but you need also export `webworker.WebWorkerTemplatePlugin` and `optimize.LimitChunkCountPlugin` at here

Reply:
I understand your point now, and I've made the necessary changes.